debezium-embedded/src/main/java/io/debezium/embedded/EmbeddedEngine.java

```diff
@@ -852,6 +854,11 @@ public Map<String, String> configs() {
                                 int maxRetries = getErrorsMaxRetries();
                                 LOGGER.info("Retriable exception thrown, connector will be restarted; errors.max.retries={}", maxRetries, e);
                                 if (maxRetries != 0) {
+                                    if (maxRetries < DEFAULT_ERROR_MAX_RETRIES) {
+                                        LOGGER.error("Can't start the connector, no attempts made to retry; stopping connector...", e);
```

Comment:
the message is misleading, there is no attempt on starting the connector. I don't think we should log anything here since it will be logged when testing the retryError handler